usr/sbin/p11sak/p11sak.c

```diff
@@ -2149,7 +2149,9 @@ static CK_RV list_ckey(CK_SESSION_HANDLE session, p11sak_kt kt, int long_print)
             printf("%s\n", label);
         }
         free(label);
+        label = NULL;
```

Comment:
Commit title: Its more to avoid a double free, not a memory leak. Maybe you can rephrase just the commit title (and shorten the title while you are at it, it overflows into the description).

Reply:
Commit message should be changed now.

Reply:
Yes, but now the PR contains only one commit. The second commit seems to be gone.....

Reply:
Should be back again.